app/core/migrations/0002_course_interestlist.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.1.13 on 2021-07-14 13:53
```

Comment:
Are you planning on squashing this into the initial migration?

Reply:
I wanted to ask about this, now that we have deployed to ADL, do we want to keep the migrations? or are fine with clearing their database on deploys? There is a command to squash migrations but the documentation suggest not deleting old ones in a "production" environment

Reply:
Yep, agreed, we've lost "total" control at this point -- we should ask ADL what they think. I'm not sure there's much state captured in the XDS db right now aside from some config and maybe a few user accounts so it should be fine.

You are right, however, that it's becoming more difficult to contain the migration scripts -- at some point we'll have to let them run.